hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -778,15 +778,14 @@ def validate_health_check_suppressions(suppressions):
 
 settings._define_setting(
     'use_coverage',
-    default=True,
+    default=False,
```

Comment:
We should default this to `not_set` or `None`, so that we can warn users with an explicit `use_coverage=False`, which will also break when we fully remove the setting.